smriprep/workflows/anatomical.py

```diff
@@ -209,11 +209,18 @@ def init_anat_preproc_wf(skull_strip_template, fs_spaces, template, debug,
 [@ants, RRID:SCR_004757]"""
     desc += '.\n' if num_t1w > 1 else ", and used as T1w-reference throughout the workflow.\n"
 
+    desc = """\
+The T1w-reference was then skull-stripped with a *Nipype* implementation of
+the `antsBrainExtraction.sh` workflow (from ANTs), using {skullstrip_tpl}
+as target template.
+""".format(skullstrip_tpl=skull_strip_template)
```

Comment:
What's our minimum Python? If it's 3.6 or higher, we can switch to [f-strings](https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals).

Reply:
I was actually thinking of that as well. We have Python +3.5 but I'd seriously consider enforcing +3.6 for that very reason.